pkg/context/context.go

```diff
@@ -24,15 +25,42 @@ type Context struct {
 	NewCommand CommandBuilder
 }
 
+type AuditableWriter struct {
```

Comment:
I hate to nit on the name but I think this name is misleading. Audit implies that it will record who performed actions so that there is accountability. This is more of a "censored" writer, who will bleep stuff out. 😀

Reply:
You're right, this needed renaming.  To that end, I've updated to `CensoredWriter`.